bigquery-flash-data-sync/.env.example

```
@@ -0,0 +1,148 @@
+# Database server port
+SALESFORCE_DB_PORT=3306
+# Name of the database/schema to connect to
+SALESFORCE_DB_NAME=salesforce_mirror
+# Database username for authentication
+SALESFORCE_DB_USER=sf_reader
+# Database password for authentication
+SALESFORCE_DB_PASSWORD=your_secure_password_here
+# Comma-separated list of tables to sync from this database
+SALESFORCE_TABLES=opportunities,contacts,accounts,leads
+
+MAX_ROW_PARSE_FAILURES=100 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)